articles/iot-hub/quickstart-send-telemetry-python.md

```diff
@@ -108,6 +108,13 @@ The simulated device application connects to a device-specific endpoint on your
     The following screenshot shows the output as the simulated device application sends telemetry to your IoT hub:
 
     ![Run the simulated device](media/quickstart-send-telemetry-python/SimulatedDevice.png)
+    
+    ### To avoid the import iothub_client error
+    The current version of the Azure IoT SDK for Python is a wrapper over [our C SDK](https://github.com/azure/azure-iot-sdk-c). It is   generated using the [Boost](https://www.boost.org/) library. Because of that, it comes with several significant limitations. See more details [here](https://github.com/Azure/azure-iot-sdk-python#important-installation-notes---dealing-with-importerror-issues)
```

Comment:
There appears to be some extra space in the second sentence here. Not a big deal though. I can fix it after merge